Math-sqrt-function.md

```diff
@@ -0,0 +1,34 @@
+# Math.sqrt() Function in Java
+**Description:** Math.sqrt() method returns the square root value of the primitive data type .If a number is less than zero, it will return NaN.
```

Comment:
``` diff
- Math.sqrt() method returns the square root value of the primitive data type .If a number is less than zero, it will return NaN.
+ `Math.sqrt()` method returns the square root value of the primitive data type .If a number is less than zero, it will return `NaN`.
```